PurchasesTests/SwiftSources/Purchasing/PurchaserInfoHelperTests.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  PurchaserInfoHelperTests.swift
+//  PurchasesTests
+//
+//  Created by RevenueCat.
+//  Copyright © 2020 Purchases. All rights reserved.
+//
+
+import Nimble
+import XCTest
+@testable import Purchases
+
+class PurchaserInfoHelperTests: XCTestCase {
+    let dateFormatter = DateFormatter()
+
+    let dict = [
+        "100_coins": [
+            [
+                "id": "72c26cc69c",
+                "is_sandbox": true,
+                "original_purchase_date": "1990-08-30T02:40:36Z",
+                "purchase_date": "2019-07-11T18:36:20Z",
+                "store": "app_store"
+            ], [
+                "id": "6229b0bef1",
+                "is_sandbox": true,
+                "original_purchase_date": "2019-11-06T03:26:15Z",
+                "purchase_date": "2019-11-06T03:26:15Z",
+                "store": "play_store"
+            ]],
+        "500_coins": [
+            [
+                "id": "d6c007ba74",
+                "is_sandbox": true,
+                "original_purchase_date": "2019-07-11T18:36:20Z",
+                "purchase_date": "2019-07-11T18:36:20Z",
+                "store": "play_store"
+            ], [
+                "id": "5b9ba226bc",
+                "is_sandbox": true,
+                "original_purchase_date": "2019-07-26T22:10:27Z",
+                "purchase_date": "2019-07-26T22:10:27Z",
+                "store": "app_store"
+            ]]
+    ]
+
+    override func setUp() {
+        dateFormatter.timeZone = TimeZone(abbreviation: "GMT")
+        dateFormatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ssZ"
+        dateFormatter.locale = Locale(identifier: "en_US_POSIX")
+    }
+
+    func testNonSubscriptionsIsCorrectlyCreated() {
+        let list: Array<Transaction> = PurchaserInfoHelper.initNonSubscriptionTransactions(with: dict, dateFormatter: dateFormatter)
```

Comment:
the type can be inferred, so I'd leave it out.
But if you do want it to be explicit, I'd replace `Array<Transaction>` -> `[Transaction]`